spec/class_game_spec.rb

```diff
@@ -0,0 +1,80 @@
+require_relative '../class_games_handler'
+require_relative '../class_author'
+require_relative '../class_game'
+require_relative '../date_validator'
+
+RSpec.describe GameHandler do
```

Comment:
- [ ] Please make sure to add a test for the method `can_be_archived?()` in the Game class. :pray:

Reply:
Thank you. I add. class_game_can_be_archive_spec.rb